etc/bin/defn-order-test.py

```diff
@@ -1,62 +0,0 @@
-# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
```

Comment:
This script was broken, it hasn't been run for a while (because it is broken) and nonone has complained so guessing its good to go.